src/dbos-executor.ts

```diff
@@ -780,12 +789,16 @@ export class DBOSExecutor implements DBOSExecutorContext {
         this.logger.error(`Cannot find communicator info for UUID ${workflowUUID}, name ${nameArr[2]}`);
         throw new DBOSNotRegisteredError(nameArr[2]);
       }
+      tempWfType = TempWorkflowType.external;
+      tempWfName = getRegisteredMethodName(commInfo.communicator);
+      tempWfClass = getRegisteredMethodClassName(commInfo.communicator);
```

Comment:
Once I fix the workflow class name recording, we're able to use `wfStatus.workflowClassName`

Reply:
Next PR!